src/main/webapp/viewTech.html

```diff
@@ -51,6 +51,11 @@
       <alert ng-show="alert" type={{alert.type}} close="closeAlert()">{{alert.msg}}</alert>
     </div>
 
+
+    <div>
+      <img ng-show="loadEndorsements" src="https://storage.googleapis.com/tech-gallery-assets/imagesLogo/Loading.gif" class="loading">
```

Comment:
Projects assets must be within the project's source-code repository and not on GCS.